### PR TITLE
Detect_lesions_s4770608

### PR DESCRIPTION
**Pull request by baibizhe**

### CODE REVIEW

**Comment by nathasha-naranpanawa** [comment]
**This is an initial inspection, no action is required at this point**

Difficulty: Normal

Readme:  OK
- no insights/discussion/interpretation of results
- some training loss and accuracy(mAP) plots shown, but not sure what they mean
- no detection results shown (**embedded images are not appearing in the readme**)

Commit messages:  Poor
- only 10 commits made within 5 days
- limited descriptions

Code:
- No hyperparameter tuning or attempts to improve performance
- Some comments are present, both in English and Chinese(?)

Functionality/Performance:
- all average metrics are below expected
- no classification accuracy given

**Comment by shakes76** [comment]
# Marking
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate design and implementation
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage, results not visualisaed -1
Module present
Commenting sometimes -1
No Data leakage
Difficulty: Normal -5
## Commit Log
Meaningful commit messages
Progressive commits used, but only 10, minimal logs -1
## Documentation
ReadMe acceptable, broken image links -1
Model/technical explanation
Good Description and Comments 
Markdown used and PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required
Request Description not given -2

**Comment by shakes76** [comment]
Please remove gitignore file for mergeing, does not affect grade only merge.

**Comment by wangzhaomxy** [comment]
No feedback attempt and no feedback marks lost.
